src/backend/vulkan/src/physical_device.rs

```diff
@@ -191,54 +702,14 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
             return Err(CreationError::MissingFeature);
         }
 
-        let imageless_framebuffers = self.api_version >= Version::V1_2
-            || self.supports_extension(vk::KhrImagelessFramebufferFn::name());
-
-        let mut enabled_features = map_device_features(requested_features, imageless_framebuffers);
         let enabled_extensions = {
-            let mut requested_extensions: Vec<&'static CStr> = Vec::new();
```

Comment:
Moved to `PhysicalDeviceInfo::get_required_extensions`, `ExtSamplerFilterMinmaxFn` is now excluded if we're >= 1.2